rc/lsp.kak

```diff
@@ -286,6 +286,7 @@ buffile  = \"${kak_buffile}\"
 filetype = \"${kak_opt_filetype}\"
 version  = ${kak_timestamp:-0}
 method   = \"textDocument/didChange\"
+hook     = true
```

Comment:
OK. Feels like a bug in Kakoune.

Reply:
Created https://github.com/mawww/kakoune/issues/4723 for it (and https://github.com/mawww/kakoune/issues/4724 for a more official detection method)